src/indra_cogex/apps/templates/home.html

```diff
@@ -42,7 +42,13 @@ <h2><span class="count-{{ key }}" data-to="{{ count }}" data-time="1000"></span>
     <div class="col" style="margin-bottom: 1em">
         {% set count, suffix = format_number(counter[key]) %}
         <span class="count-{{ key }}" data-to="{{ count }}" data-time="1000"></span>{{ suffix }}<br/>
-        <small>{{ label|replace(" ", "&nbsp;")|safe }}</small>
+        <small>
```

Comment:
I added this so it's really easy to see which key is missing a corresponding label, but if we would rather have blank space when something is missing, we can revert this change.